### DISCUSSION TITLE
Multiple/dynamic root folder mappings

### DISCUSSION THREAD

**Discussion by cosmosxcii** [sct/overseerr#1062] [Help] [answered]
Sorry if I word this poorly,

My current setup is to have multiple requestrr bots which all point to separate root folders within Ombi, I have Discord channels for people to request "Movies" or "Kids Movies" etc and it will automatically place them in the right root folder since Requestrr/Ombi setup lets you choose the root folder. 

Within Overseerr. Is there a way to fulfil something similar? Hopefully I'm not missing something painfully obvious here.

**Comment by samwiseg0** [comment, marked as answer]
Overseerr supports setting root folders per request. You would need to ask over in Requestrr to verify on how this should be setup with Overseerr.
